docs/jitsi.md

```diff
@@ -0,0 +1,47 @@
+# Jitsi in Riot
+
+Riot uses [Jitsi](https://jitsi.org/) for conference calls, which provides options for
+self-hosting your own server and supports most major platforms.
+
+1:1 calls, or calls between you and one other person, do not use Jitsi. Instead, those
+calls work through a TURN server configured on your respective homeservers.
+
+There's a number of ways to start a Jitsi call: the easiest way is to click on the 
+voice or video buttons near the message composer in a room with more than 2 people. This
+will add a Jitsi widget which allows anyone in the room to join.
+
+Integration managers (available through the 4 squares in the top right of the room) may
+provide their own approaches for adding Jitsi widgets, though these are now considered
+legacy and should only be used in specific circumstances.
+
+## Configuring Riot to use your self-hosted Jitsi server
+
+Riot will use the Jitsi server that is embedded in the widget, even if it is not the
+one you configured. This is because conference calls must be held on a single Jitsi
+server and cannot be split over multiple servers.
+
+However, you can configure Riot to *start* a conference with your Jitsi server by adding
+to your [config](./config.md) the following:
+```json
+{
+  "jitsi": {
+    "preferredDomain": "your.jitsi.example.org"
+  }
+}
+```
+
+The default is `jitsi.riot.im` (a free service offered by Riot), and the demo site for
+Jitsi uses `meet.jit.si` (also free).
+
+Once you've applied the config change, refresh Riot and press the call button. This
+should start a new conference on your Jitsi server. 
+
+**Note**: The widget URL will point to a `jitsi.html` page hosted by Riot. The Jitsi
+domain will appear later in the URL as a configuration parameter.
+
+## Mobile app support
+
+Currently the Riot mobile apps do not support custom Jitsi servers and will instead
+use the default `jitsi.riot.im` server. When users on the mobile apps join the call,
+they will be joining a different conference which has the same name, but not the same
+participants.
```

Comment:
"This is a bug, which needs to be fixed."

Reply:
it's not on any roadmap to be fixed right now though (aiui), so it's important to mention as this has been biting users pretty badly

Reply:
i’m not asking you to remove the explanation, just to spell out that it’s a bug, irrespective of whether it’s on a roadmap or not.

separately it’s obvious that we need to fix it, and as you know, it is on riot/mobile’s radar.